Assets/Scripts/Enemy/EnemyController.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class EnemyController : MonoBehaviour
+{
+    private float elapsedTime;
+    public float distanceToPlayer;
+    private GameObject player;
+    public NavMeshAgent agent;
+    public Vector3 destPos;
```

Comment:
```suggestion
    [HideInInspector]
    public Vector3 destPos;
```
Deze flag zorgt ervoor dat een public variabele niet zichtbaar is in de inspector